packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -100,9 +100,8 @@ abstract class FlutterCommand extends Command {
   // This is a field so that you can modify the value for testing.
   Validator projectRootValidator = () {
     if (!FileSystemEntity.isFileSync('pubspec.yaml')) {
-      printError('Error: No pubspec.yaml file found.\n'
-        'This command should be run from the root of your Flutter project.\n'
-        'Do not run this command from the root of your git clone of Flutter.');
+      printError('Error: No pubspec.yaml file found. This command should be run '
```

Comment:
Why remove the last line? It's helpful for people who think "your Flutter project" might mean "the flutter project" i.e. "your clone of the flutter project".